src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -188,7 +188,7 @@ def launch(
         port_id=BEACON_HTTP_PORT_ID,
         extract={
             "enr": ".data.enr",
-            "multiaddr": ".data.discovery_addresses[0]",
+            "multiaddr": ".data.p2p_addresses[0]",
```

Comment:
@barnabasbusa this seems like the actual fix; the discovery_address is a udp address while the p2p_address is the tcp address; changing this seems to have fixed the following configuration for me at least

```yaml
participants:
  - el_client_type: geth
    cl_client_type: lighthouse
  - el_client_type: geth
    cl_client_type: teku
  - el_client_type: geth
    cl_client_type: lighthouse
additional_services:
  - dora
```

the teku cli expects TCP in its examples; plus the other outputs of `/eth/v1/node/peers` seems to be last connecting via TCP as well

wonder why this all works on docker; maybe k8s is bottlenecked on bootnodes while docker isnt'?